Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/Use Case 1 - Paraoanu Lab - Cryoscope/readme.md

```diff
@@ -0,0 +1,221 @@
+# Cryoscope
+
+*Demonstrated on the experiment of Marko Kuzmanovic and Aidar Sultanov in the Lab of Prof. Sorin Paraoanu in Aalto University, 
+only two days after connecting the OPX to the quantum device.*
+
+*Important note: the code in this folder is the exact code that was used for running this measurement*
+*and it is tailored for a very specific setup and software environment. Thus, the code is here only for insipiration.*
+
+The goal of this use-case is to implement a cryoscope measurement as described in [[1]](#1). 
+The idea of the method is to measure the step response of the flux line using the qubit phase measured in a Ramsey-like experiment.
+The possibly distorted response can then be pre-corrected by designing suitable IIR and FIR filters.
+
+
+## 1. Experimental set-up and required calibrations
+
+
+<img align="right" src="Single Flux Tunable Transmon Setup.PNG" alt="drawing" width="400"/>
+
+### 1.1 Experimental set-up
+A standard flux tunable transmon coupled to a readout resonator set-up is used and the general scheme is represented on the figure on the right.
+
+The qubit is controlled with IQ signals generated by the OPX and upconverted with an external local oscilaltor using a previously calibrated IQ mixer.
+The readout resonator is pumped with a baseband tone generated by the OPX and upconverted with another external local oscillator using a single sideband mixer.

```

Comment:
I usually associated baseband with DC signal, like when we do digital demodulation we bring the signal at IF to baseband in a readout

Reply:
I see, how would you call the IF signal coming out of the OPX? Just  "with an intermediate frequency generated"?